### PR TITLE
Hotfix/add random ip generate wg interface #000

### PR DESCRIPTION
**Pull request by mrtrkmn**
- [x] Get random ip from ippool for initializing VPN endpoint for red and blue team
- [x] Port map given range of ports (-to be able to have different games-) with host
- [x] Initialize VPN Enpoints (red and blue team interfaces inside the machine)
- [x] Enable access with port mapping to VPN endpoints which are initialized inside NAP VM
- [x] Add some functions into goroutines 

running-vpn-interface

### CODE REVIEW

**Comment by mrtrkmn** [comment]
There are still some things needs to be done of course. We can discuss on the changes if you are not sure what I did.

File: game/environment.go [hunk 1]
```diff
@@ -172,109 +176,155 @@ func (g *environment) Close() error {
...
 	// bridge name will be same with event tag
 	bridgeName := tag
+	var mainErr error
+	// todo: scenarios should be taken from a resource such as postresql / mongodb !
 	selectedScenario := TemporaryScenariosPlaceHolder[scenarioNo]
+
 	numNetworks := len(selectedScenario.Networks)
 	log.Info().Msgf("Setting openvswitch bridge %s", bridgeName)
+	var waitGroup sync.WaitGroup
+	waitGroup.Add(2)
```
Comment (rasmim) [inline game/environment.go:194]: Why waitGroup.Add(2) ?

Comment (mrtrkmn) [reply game/environment.go:194]: Leftovers just delete them :)

**Comment by rasmim** [comment]
It was merged to Master :muscle:

Cheers,
Vlad

**Pull request closed without merging**